tests/Tests/200__monitor_and_manage/220__grafana/220__test_verify_port_forwarding_grafana.robot

```diff
@@ -0,0 +1,48 @@
+*** Settings ***
+Documentation   Verify that MT-SRE can connect to Grafana using port-forwarding
+...
+...             = Variables =
+...             | PROC             | Required |        Is used  to get the process id|
+
+Library        OperatingSystem
+Library        Process
+Library        SeleniumLibrary
+
+
+*** Variables ***
+${PROC}     none
+
+
+*** Test Cases ***
+Verify that MT-SRE can connect to Grafana using port-forwarding
+    [Documentation]  Verify Grafana webiste is using port-forwarding
+    [Tags]   Tier2
+     ...     ODS-754
+
+    # Port Forwarding
+    ${PROC}    Start Process    oc -n redhat-ods-monitoring port-forward $(oc get pods -n redhat-ods-monitoring | grep grafana | awk '{print $1}' | head -n 1) 3001  shell=True  # robocop: disable
```

Comment:
It could be an test keyword `Enable Access To Grafana Using OpenShift Port Forwarding`